experimental/PropertyDDS/packages/property-shared-tree-interop/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluid-experimental/property-shared-tree-interop",
-	"version": "2.0.0-rc.2.0.0",
+	"version": "2.0.0-rc.1.0.0",
 	"description": "Utilities for migration from PropertyDDS to the new SharedTree DDS",
 	"homepage": "https://fluidframework.com",
```

Comment:
This is now a CJS build. I assume that is not the intent. Otherwise, why keep build:esnext?